code/drasil-example/Drasil/DblPendulum/DataDefs.hs

```diff
@@ -60,7 +60,7 @@ frequencyDDQD :: QDefinition
 frequencyDDQD = mkQuantDef QP.frequency frequencyDDEqn
 
 frequencyDDEqn :: Expr
-frequencyDDEqn = 1 / sy QP.period
+frequencyDDEqn = dbl 1 $/ sy QP.period
```

Comment:
The more I see things like this, the more I think we need an `Exact` constructor that maps integers to reals.

Reply:
I like the idea of `Exact`! I'll make it! Thank you.

Reply:
Now, it and similar equations use `exactDbl` to construct them, leaving us with prettier looking equations when printed, but the more appropriate variants using `1.0` for the code generation. Thank you!